ichnaea/log.py

```diff
@@ -59,6 +62,15 @@ def configure_logging():
         "root": {"handlers": handlers, "level": "WARNING"},
     }
 
+    if local_dev_env:
+        # In the local dev environment, add markus as a logger so that we can
+        # see metrics in the logs
+        logging_config["loggers"]["markus"] = {
+            "propagate": False,
+            "handlers": handlers,
+            "level": level,
```

Comment:
Markus doesn't log anything unless the logging backend is configured, so I think I'll tweak this so it's not optional.